src/commands/push-branches.js

```diff
@@ -13,24 +13,36 @@ exports.handler = async (_, state) =>
 			})
 			const repoBranch = incrementSuffix(branches, repo.localBranch)
 
-			await logger.logPromise(
-				git.push({
-					repository: 'origin',
-					refspec: `${repo.localBranch}:${repoBranch}`,
-					workingDirectory: repo.clone,
-				}),
-				`pushing ${styles.branch(repo.localBranch)} to ${styles.repo(
-					`${repo.owner}/${repo.name}`,
-				)}${
-					repoBranch !== repo.localBranch
-						? ` (as ${styles.branch(repoBranch)} because ${styles.branch(
-								repo.localBranch,
-						  )} already exists)`
-						: ''
-				}`,
-			)
+			const newCommits = await git.cherry({
+				workingDirectory: repo.clone,
+				origin: 'origin',
+				branch: repo.localBranch,
```

Comment:
correct options from https://github.com/Financial-Times/tooling-helpers/pull/86:

```suggestion
				upstream: 'origin',
				head: repo.localBranch,
```

Reply:
thank you for spotting this!